addons/pinniped/config-controller/controllers/v3_controller.go

```diff
@@ -46,13 +46,13 @@ func (c *PinnipedV3Controller) SetupWithManager(manager ctrl.Manager) error {
 		).
 		Complete(c)
 	if err != nil {
-		c.Log.Error(err, "error creating pinniped config controller")
+		c.Log.Error(err, "error creating")
```

Comment:
should we say `"error creating controller"` like we have for v1a1?

Reply:
updated!